README.md

```diff
@@ -254,6 +257,9 @@ IS&T International Symposium on Electronic Imaging 2019. Computational Imaging X
 - <a name="Alzugaray193dv"></a>Alzugaray, I., Chli, M.,  
 *[Asynchronous Multi-Hypothesis Tracking of Features with Event Cameras](https://doi.org/10.3929/ethz-b-000360434)*,  
 Int. Conf. 3D Vision (3DV), 2019. [PDF](https://doi.org/10.3929/ethz-b-000360434), [YouTube](https://youtu.be/eguV_AIbteU)
+- <a name="LinaresBarrancoA19access"></a>Linares-Barranco, A., Perez-Pena, F., Moeys, D., Gomez-Rodriguez, F., Jimenez-Moreno, G., Delbruck, T. 
```

Comment:
Same thing here. Need two white spaces at the end of the line to create a line break.